motion-1/Overview.bs

```diff
@@ -163,19 +235,235 @@ Media: visual
+
+See the section <a href="#offset-processing">“Offset processing”</a> for how to process an 'offset-distance'.
+
+<div class='example'>
+This example shows a way to align elements within the polar coordinate system using 'offset-path', 'offset-distance'.
+<pre class="lang-html">
+	&lt;body>
+		&lt;div id="circle1"&lt;/div>
+		&lt;div id="circle2"&lt;/div>
+		&lt;div id="circle3"&lt;/div>
+	&lt;/body>
+	&lt;style>
+		#circle1 {
+			offset-path: 0deg;
+			offset-distance: 50%;
+		}
+		#circle2 {
+			offset-path: 90deg;
+			offset-distance: 20%;
+		}
+		#circle3 {
+			offset-path: 225deg;
+			offset-distance: 100%;
+		}
+	&lt;/style>
+</pre>
+<div class=figure> 
+	<img alt="An image of three elements positioned to polar coordinates" src="images/simple_offset_position.png" style="width: 300px;"/>
+	<figcaption>An example of positioning element in polar coordinates</figcaption>
+</div>
+</div>
 
-<dl dfn-for="motion-offset" dfn-type="value">
-  <dt><<length-percentage>></dt>
-  <dd>The distance from the <a>initial position</a> of the shape or path to the position of the element. Percentages are relative to the total path length including all sub-paths.</dd>
+<h3 id="offset-position-property">Define the initial position of the path: The 'offset-position' property</h3>
+<pre class='propdef'>
+Name: offset-position
+Applies to: all elements
+Value: auto | <<position>>
+Initial: auto
+Media: visual
+Inherited: no
+Percentages: Refer to the size of containing block
+Animatable: as <a href="https://www.w3.org/TR/css3-transitions/#animtype-lpcalc">position</a>
+</pre>
+Specifies the initial position of the path.
+
+Values are defined as follows:
+<dl dfn-for="offset-position" dfn-type="value">
+<dt><var>'auto'</var></dt>
+<dd>Computes to the current position of the element.</dd>
+
+<dt><<position>></dt>
+<dd>The <<position>> represents the horizontal position and the vertical position 
+	as specified in CSS Backgrounds [[CSS3-BACKGROUND]].
+
+	Values of <<position>> are defined like below:
+	<dl>
+		<dt><<percentage>></dt>
+		<dd>A percentage for the horizontal offset is relative to the width of 
+		the containing block.
+		A percentage for the vertical offset is relative to the height of 
+		the containing block.
+		</dd>
+
+		<dt><<length>></dt>
+		<dd>A length value gives the fixed length as an offset.
+		The offset is from the top left corner of the containing block.
+		</dd>
+
+		<dt><dfn>top</dfn></dt>
+		<dd>Computes to 0% for the vertical position.</dd>
+
+		<dt><dfn>right</dfn></dt>
+		<dd>Computes to 100% for the horizontal position.</dd>
+
+		<dt><dfn>bottom</dfn></dt>
+		<dd>Computes to 100% for the vertical position.</dd>
+
+		<dt><dfn>left</dfn></dt>
+		<dd>Computes to 0% for the horizontal position.</dd>
+
+		<dt><dfn>center</dfn></dt>
+		<dd>Computes to 50% (left 50%) for the horizontal position 
+		if the horizontal position is not otherwise specified, 
+		or 50% (top 50%) for the vertical position if it is.</dd>
+	</dl>
+</dd>
+</dl>
```

Comment:
I put the duplicated definition because some parts of the definition of << position >> in CSS3-BACKGROUND are different in here. 
For specifying the horizontal offset or the vertical offset with << percentage >>, the reference distance would be changed from (width/height of background positioning area - width/height of background image) to width/height of the containing block.

Then I will mention the changes in the definition like above and remove the definition.